### ISSUE TITLE
Events list just show the first day of multiple "whole-day" events

### ISSUE THREAD

**Issue by meaz** [SimpleMobileTools/Simple-Calendar#742]
Create an event that lasts on several days, let's say the 2nd and the 3rd of January.
Choose that there is no time for this event but instead it lasts the whole day
Go on the "events-list" view.
You will see an event on the 2nd of January, but nothing on the 3rd of January.

**Comment by jtj9817** [comment]
@tibbi Hey, can I also work on this bug? I don't know if #751 is resolved or not yet but I want to work on this bug too so I can get to know the project better.

**Comment by tibbi** [comment]
Well it needs some bigger UI update and Im not yet sure how should it look like, so this probably isnt the best issue to work on.

**Comment by doctaweeks** [comment]
I may be missing some context but can this be fixed without #751? Correctly displaying a multi-day event is a pretty standard feature that should work.

**Comment by jtj9817** [comment]
@doctaweeks Yeah I think this can be fixed without #751. If anything, I can check again to see if the layout changes from #751 doesn't apply correctly when this Issue gets an attempted fix. @tibbi If it needs a UI update and there's no concrete plans yet on how to implement it, I can work on it too if possible and release it incrementally to make sure that nothing breaks.

**Comment by tibbi** [comment]
this isnt related to #751 at all. I just want to do some bigger UI updates to list views, I dont really like the current versions. Neither in the app, nor on the widget.

**Comment by jtj9817** [comment]
@tibbi Do you mind if we coordinate the implementation for these UI updates? Maybe start an Issue for it (with the implementation plans laid out) and I'll start implementing some of the plausible ideas.
